cli/cmd/cluster/certificate-rotator.go

```diff
@@ -99,7 +99,7 @@ func rotateControlPlaneCerts(contextLogger *log.Entry, cc clusterConfig) error {
 
 	contextLogger.Log(log.InfoLevel, "Applying a controlplane update with the new CA")
 
-	if err := c.upgradeControlPlane(contextLogger, kubeconfig); err != nil {
+	if err := c.upgradeControlPlane(contextLogger, kubeconfig, true); err != nil {
```

Comment:
Maybe it makes sense to add an option here to skip rotating the kubelet?

Reply:
>Maybe it makes sense to add an option here to skip rotating the kubelet?

Any specific reason for it? I guess that would be a feature request?

Reply:
Maybe this for better readability? Boolean parameters in functions are smells, so maybe at least we can name them :)
```suggestion
        upgradeKubelets := true`
        
	if err := c.upgradeControlPlane(contextLogger, kubeconfig, upgradeKubelets); err != nil {
```

Reply:
Yeah, I don't think skipping rotating the kubelet makes sense in this PR.

Sure, I'll add a bool variable.